### PR TITLE
Judging admin fixes

### PR DESCRIPTION
**Pull request by ianmah**

### CODE REVIEW

File: src/containers/JudgingPanel.js [hunk 1]
```diff
@@ -305,16 +338,31 @@ export default () => {
...
+            />
+            <P>Toggle Projects/Grades</P>
+          </Column>
+
+          <Column>
+            <ToggleSwitch
+              checked={toggle.filterDisqualify}
+              onChange={() => setToggle({ filterDisqualify: !toggle.filterDisqualify })}
+            />
+            <P>Filter disqualfied</P>
```
Comment (jackyzha0) [inline src/containers/JudgingPanel.js:355]:
```suggestion
            <P>Filter disqualified</P>
```

File: src/pages/Submission.js [hunk 2]
```diff
@@ -25,7 +25,16 @@ export default () => {
     const submittedProjectRef = d.submittedProject
     if (!!submittedProjectRef) {
       const submission = await getSubmission(submittedProjectRef)
-      setSubmission(submission)
+
+      Object.keys(submission.grades).forEach(id => {
+        if (submission.grades[id].removed) {
+          delete submission.grades[id]
+        }
+      })
```
Comment (jackyzha0) [inline src/pages/Submission.js:33]: why not filter here?

Comment (ianmah) [reply src/pages/Submission.js:33]: I think we need to keep it as an object not an array

Comment (jackyzha0) [reply src/pages/Submission.js:33]: ah forgot this was an object, ok

**Review by jackyzha0 — APPROVED** [review]
🏺 nice

**Pull request merged**